jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Attachment.java

```diff
@@ -22,6 +22,7 @@
  *******************************************************************************/
 package com.ibm.tools.attach.target;
 
+import com.ibm.oti.vm.VM;
```

Comment:
Only one place benefits from this import; consider removing it or removing the redundant qualification where it is referenced.